### PR TITLE
JAX-WS: HttpConduitProperties test is refactored

### LINKED ISSUE #24774: HttpConduitProperties test is refactored
Test was prone to errors and was inefficient. It's refactored in a way that reconfiguration changes are pulled from a file included in server.xml. That way, the requirement of server or application restart eliminated. This solution is aimed to fix timing issues between file copy and test run processes.

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #24774

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java [hunk 1]
```diff
@@ -65,6 +59,8 @@ public class HttpConduitPropertiesTest {
     // WAIT TIME to be used to wait for copying ibm-ws-bnd.xml to complete before invoking Web Service
     private static int BND_FILE_COPY_WAITTIME = 1000;
 
+    private static int CONFIG_REFRESH_COUNT = 0;
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java:62]: Is this variable being used any more?

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**